torch/export/exported_program.py

```diff
@@ -283,6 +284,17 @@ def _split_decomp_table_to_cia_and_python_decomp(
     return cia_ops_to_callable, decomp_table
 
 
+def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
+    """
+    This is an official decomposition table which contains decomposition of
```

Comment:
Cut `official`? You can say "default." Or:

This returns a table of decompositions for all aten operators to core aten operators. Use this API...